src/components/Editor.jsx

```diff
@@ -1,5 +1,18 @@
 /* global WPGRAPHQL_IDE_DATA */
 import { GraphiQL } from 'graphiql';
+import {
+	ToolbarButton,
+	PrettifyIcon,
+	usePrettifyEditors,
+	useCopyQuery,
+	useMergeQuery,
+	MergeIcon,
+	CopyIcon,
+	useEditorContext
+} from '@graphiql/react';
+
+// Assuming wp.hooks is globally available through WordPress's script enqueueing mechanism.
+const { applyFilters } = wp.hooks;
```

Comment:
We can use `@wordpress/hooks` here so that this can eventually be ran outside of WordPress and also signal to `@wordpress/scripts` to add this dependency to *.asset.php.